acquire/acquire.py

```diff
@@ -1292,6 +1294,31 @@ def get_specs(cls, cli_args):
         return [(path_selector, hash_funcs) for path_selector in path_selectors]
 
 
+@register_module("--handles")
+@module_arg("--file-handle-type", action="append", help="Only collect file handles")
+@local_module
+class OpenHandles(Module):
+    DESC = "Open handles"
+
+    @classmethod
+    def run(cls, target: Target, cli_args: dict[str, any], collector: Collector):
+        log.info("*** Acquiring open handles")
+
+        file_only = cli_args.file
```

Comment:
I'm confused by these arguments. You specify `--file-handle-type` as argument for the module, with action append, but check for the `file` argument, and treat it like a boolean?

Reply:
Yep, forgot about that one... Was in doubting how to tackle it. Fixed it now :)